### ISSUE TITLE
Throw better error and/or handle fxa-content-server METRICS_DOC_URL dependency differently

### ISSUE THREAD

**Issue by data-sync-user** [mozilla/fxa#12227]
A URL update to an ecosystem-docs page caused a `TypeError` to appear with a very misleading stack trace:

```
...
✓ node - routeLogging - it logs a json blob if log format is not dev_fxa

node
Total: [✓✓✓✓✓✓✓✓✓✓✓✓✓✓✓✓✓✓✓✓✓✓✓✓✓✓✓✓✓✓✓✓✓✓✓✓✓✓✓✓] 764/772
Passed: 764  Failed: 0    Skipped: 0

TypeError: Cannot read properties of undefined (reading 'browserName')
    at Pretty._abbreviateEnvironment (/home/circleci/project/node_modules/intern/lib/reporters/Pretty.js:251:40)
    at Pretty._drawSessionReport (/home/circleci/project/node_modules/intern/lib/reporters/Pretty.js:240:38)
    at Pretty._render (/home/circleci/project/node_modules/intern/lib/reporters/Pretty.js:289:26)
    at Pretty.runEnd (/home/circleci/project/node_modules/intern/lib/reporters/Pretty.js:100:18)
    at /home/circleci/project/node_modules/intern/lib/reporters/Reporter.js:83:50
    at Node.BaseExecutor.emit (/home/circleci/project/node_modules/intern/lib/executors/Executor.js:144:62)
    at /home/circleci/project/node_modules/intern/lib/executors/Executor.js:335:77
    at /home/circleci/project/node_modules/@theintern/common/index.js:2:34567
    at /home/circleci/project/node_modules/@theintern/common/index.js:2:34861 {
  reported: true
}
```

Turns out it was the test that ran next, which is [the suite](https://github.com/mozilla/fxa/blob/main/packages/fxa-content-server/tests/server/validation.js#L12) in `validator.js`. Updating `METRICS_DOCS_URL` to the correct URL fixed the problem (and, the functions in that Pretty.js file doesn't even get hit when it’s a valid URL, making debugging this not a great experience).

As a minimum, if we can at least throw a try/catch around this and throw an error that says the page has 404’d if it can’t be found, it would be a major improvement. It might also be worth looking to see if we reference ecosystem-doc links anywhere else to do the same there, and potentially evaluate the need for these tests. Maybe there’s also a better way to reference these URLs in FxA, like moving them into a single shared file/constants object, and creating a PR template in ecosystem-platform to link to that file as a heads-up that if any URLs were updated they need to also be updated on the FxA side.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXA-4746)

**Comment by data-sync-user** [comment]
➤ Wil Clouser commented:

We ran into this problem when a 3rd party’s site was down and one of the links in our privacy notice started 404ing.  After a bunch of distraction and investigating we disabled it and it’s apparently still disabled ( https://github.com/mozilla/fxa/blob/main/packages/fxa-content-server/tests/server/routes.js#L11 ).  

We should be very deliberate about the scope of our unit tests and I believe that unit tests should be self contained.  Any unit test accessing remote URLs, including the one using METRICS_DOCS_URL, should be deleted.
